modules/apps/remote-app/remote-app-service/src/main/java/com/liferay/remote/app/service/impl/RemoteAppEntryLocalServiceImpl.java

```diff
@@ -195,6 +195,46 @@ public RemoteAppEntry addIFrameRemoteAppEntry(
 		return _startWorkflowInstance(userId, remoteAppEntry);
 	}
 
+	@Indexable(type = IndexableType.REINDEX)
+	@Override
+	public RemoteAppEntry addOrUpdateCustomElementRemoteAppEntry(
+			long userId, String customElementCSSURLs,
+			String customElementHTMLElementName, String customElementURLs,
+			String description, String externalReferenceCode,
```

Comment:
So sorry... I missed this in my first review. The externalReferenceCode param is always the first parameter as it is an identifier of the resource. Brian wanted this way for all the entities that already have it

Reply:
No problem!
I changed to be first argument.